opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -501,6 +530,50 @@ def use_span(
             end_on_exit: Whether to end the span automatically when leaving the
                 context manager.
         """
+
+
+class DefaultTracer(Tracer):
+    """The default Tracer that is used when no Tracer implementation is available.
```

Comment:
```suggestion
    """The default Tracer, used when no Tracer implementation is available.
```

Just to make it fit under 79 chars. Black doesn't care about comment line length and we had to disable the general purpose line length check to use black, so we have to format these by hand.